app/web/cypress/e2e/modelling-functionality/create-component.cy.ts

```diff
@@ -30,19 +31,22 @@ Cypress._.times(SI_CYPRESS_MULTIPLIER, () => {
       // Give time to redirect onto the new changeset
       cy.url().should('not.include', 'head', { timeout: 10000 });
 
-      // Find the AWS Region
-      cy.get('div[class="tree-node"]', { timeout: 30000 }).contains('Region').as('region');
+      // Create a region component
+      cy.createComponent("region");
 
-      // Find the canvas to get a location to drag to
-      cy.get('canvas').first().as('konvaStage');
+      // // Find the AWS Region
+      // cy.get('div[class="tree-node"]', { timeout: 30000 }).contains('Region').as('region');
 
-      // drag to the canvas
-      cy.dragTo('@region', '@konvaStage');
+      // // Find the canvas to get a location to drag to
+      // cy.get('canvas').first().as('konvaStage');
 
-      cy.wait(5000);
+      // // drag to the canvas
+      // cy.dragTo('@region', '@konvaStage');
 
-      //check to make sure a component has been added to the outliner
-      cy.get('[class="component-outline-node"]', { timeout: 30000 }).contains('Region', { timeout: 30000 }).should('be.visible');
+      // cy.wait(5000);
+
+      // //check to make sure a component has been added to the outliner
```

Comment:
nit: I think it's totally cool to remove the original lines, they aren't really providing value anymore after you got it working! :)